### PR TITLE
Bump KA version to include token rotation fix

### PR DESCRIPTION
**Pull request by dougszumski**
(cherry picked from commit eadcdedb74971f6b8bcacc85d0a8487782246cb5)

This has not been applied yet.

### CODE REVIEW

**Comment by markgoddard** [comment]
Merge conflict

**Comment by markgoddard** [comment]
Can we instead do this by merging alaska-alt-1 into alaska-prod?

**Comment by dougszumski** [comment]
No problem, will wait for alt-1 change to be merged before updating.

**Comment by dougszumski** [comment]
This change has been applied to prod.

File: etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% raw %}
```
Comment (markgoddard) [inline etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml]: I think you'll need to rename this to sv-b16-u21.

Comment (dougszumski) [reply etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml]: doh

**Review by markgoddard — APPROVED** [review]

**Pull request merged**